deps/stlport/stl/_algo.h

```diff
@@ -22,6 +22,9 @@
  * modified is included with the above copyright notice.
  *
  */
+/*
+ * Code has been modified to add 'clamp' function from C++ 2017. - Duncan Frost 2021
```

Comment:
Did you mean to write that name here?

Reply:
I was just trying to follow the style of the file but I'm also thinking that perhaps this isn't how best to show it anyway since it looks like this is an original sgi file so i should just add an extra line under the other authors